EIPS/eip-8000.md

```diff
@@ -0,0 +1,76 @@
+---
+eip: 8000
```

Comment:
```suggestion
eip: 7592
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
EIP number and filename has been updated, thank you for the suggestion.